src/lib.rs

```diff
@@ -434,61 +501,61 @@ make_color! {
     ///Linear luminance.
     Luma {
         ///Linear luminance.
-        y(luma: T)[alpha: T] => new;
+        y(luma: T)[alpha: T] => with_wp;
```

Comment:
This works for now and will require more changes for the Rgb Variants. Once we have that implemented, we will have a better idea how complicated color enum gets and maybe think about it then?